website/content/en/docs/component-guides/profiles.md

```diff
@@ -55,7 +59,11 @@ After installing Kubeflow on AWS with one of the available [deployment options](
    export PROFILE_CONTROLLER_POLICY_NAME=<the name of the profile controller policy to be created>
    ```
 
-2. Create an IAM policy using the [IAM Profile controller policy](https://github.com/awslabs/kubeflow-manifests/blob/main/awsconfigs/infra_configs/iam_profile_controller_policy.json) file.
+### 2. Configure the IAM Profile controller
```

Comment:
suggest reword: Configure the Profile Controller or Configure the IAM permissions for Profile Controller

Reply:
done